pkg/compiler/codegen.go

```diff
@@ -1349,7 +1349,7 @@ func (c *codegen) isCallExprSyscall(e ast.Expr) bool {
 // Thus, we use the following approach:
 // 1. Throwed exception is saved in a static field X, static fields Y and it is set to true.
 // 2. For each defer local there is a dedicated local variable which is set to 1 if `defer` statement
-//    is encountered during an actual execution.
+// is encountered during an actual execution.
```

Comment:
Shouldn't we make
```patch
-1...
+ 1...
```
(because lists are to be indented) And reindent it appropriately?